apps/ejabberd/src/node_push.erl

```diff
@@ -207,20 +208,16 @@ is_allowed_to_publish(PublishModel, Affiliation) ->
 parse_form(undefined) ->
     #{};
 parse_form(Form) ->
-    IsForm = ?NS_XDATA == exml_query:attr(Form, <<"xmlns">>),
-    IsSubmit = <<"submit">> == exml_query:attr(Form, <<"type">>),
```

Comment:
@astro @michalwski I didn't see that there is a mistake in examples from [XEP-0357](https://xmpp.org/extensions/xep-0357.html#publishing). Because of that we may skip this check but not this way in my opinion. My proposition is that we leave this check active, but we can default to `submit` form `type` in case this attribute is missing. This way, the current implementations that don't send this attribute will work and the code will reject wrong form types like `cancel`. What do you think about it @michalwski ?